### DISCUSSION TITLE
Language shows up in statistics despite omitting it in ReadMe

### DISCUSSION THREAD

**Discussion by MatthewTess** [anuraghazra/github-readme-stats#2896] [Q&A] [answered]
I tried variations for the portion of the readme which displays language usage statistics, and no matter the approach, when I hide language and select C#, it still shows C# on my user homepage. Usually, without hiding a language, C# shows as 95% usage, but when omitting C#,  it cuts usage by half but not entirely somehow. When using a third-party website to determine usage, C# is displayed at a usage rate of only 6.5%. I even deleted the entire repo for CSE155 and double-checked to make sure C# is not used elsewhere, and when returning to my profile home page it still shows C# at majority usage. Gitattributes special-vendored-path/* linguist-vendored also does nothing when placed within the desired repo. This is my first time editing my username readme for my homepage so I am a bit of a novice, but I am hoping to get some pointers as to what is going wrong.

![Screenshot (6320)](https://github.com/anuraghazra/github-readme-stats/assets/95320458/c963ad15-4386-46da-a4bc-4cda94865a51)
![Screenshot (6321)](https://github.com/anuraghazra/github-readme-stats/assets/95320458/36aafb43-479d-44b0-b4a0-560f5a9841cb)
![Screenshot (6319)](https://github.com/anuraghazra/github-readme-stats/assets/95320458/b925b916-cf29-4936-ab31-48d223626f35)

**Comment by rickstaa** [comment, marked as answer]
@MatthewTess Since the API is accessed through a `https` endpoint, some characters need to be encoded using [percent encoding](https://en.wikipedia.org/wiki/URL_encoding) (see https://github.com/anuraghazra/github-readme-stats#language-card-exclusive-options), The right syntax is:

![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=matthewTess&hide=c%23)

```md
![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=matthewTess&hide=c%23)
```

**Reply by MatthewTess** [reply]
was simple, ty it works
